docs/src/ai/external-agents.md

```diff
@@ -85,6 +85,12 @@ If you'd like to bind this to a keyboard shortcut, you can do so by editing your
 ]
 ```
 
+### Authentication
+
+As of version `0.203.2` (stable) and `0.202.7` (preview), authentication to Zed's Claude Code installation is decoupled entirely from Zed's agent. That is to say, an Anthropic API key added via the [Zed Agent's settings](./llm-providers.md#anthropic) will _not_ be utilized by Claude Code for authentication and billing.
+
+To ensure you're using your billing method of choice, [open a new Claude Code thread](./agent-panel.md#new-thread) and run `/logout`. Then, run `/login`, and authenticate either via API key, or via `Log in with Claude Code` to use a Claude Pro/Max subscription.
```

Comment:
@morgankrey I think we should just tell them to run `/login`, the `/logout` step is unnecessary

Reply:
`/logout` is effectively the same as `/login`, it just shows the auth required state. It doesn't actually `/logout` because that isn't possible over the SDK.